docs/notebooks/nb_03_wavefunction.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<p>This equation is not displaying correctly on my computer</p><p>\begin{equation} &nbsp;Q = \sum_A Z_A - N_\text{el}. &nbsp;\end{equation} </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/evangelistalab/forte/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>